src/app/(app)/members/components/MemberFormBuilder.tsx

```diff
@@ -312,6 +324,7 @@ export const MemberFormBuilder = ({ member }: MemberFormBuilder) => {
                       <Button
                         data-testid="ehad_duration"
                         variant={"outline"}
+                        onClick={()=>{dateformatter(field.value)}}
```

Comment:
I'll appricate you if you keep this is variable

Reply:
fixed